src/dev/ci_setup/setup.sh

```diff
@@ -2,116 +2,14 @@
 
 set -e
 
-dir="$(pwd)"
-cacheDir="${CACHE_DIR:-"$HOME/.kibana"}"
-
-RED='\033[0;31m'
-C_RESET='\033[0m' # Reset color
-
-###
-### Since the Jenkins logging output collector doesn't look like a TTY
-### Node/Chalk and other color libs disable their color output. But Jenkins
-### can handle color fine, so this forces https://github.com/chalk/supports-color
-### to enable color support in Chalk and other related modules.
-###
-export FORCE_COLOR=1
-
-###
-### check that we seem to be in a kibana project
-###
-if [ -f "$dir/package.json" ] && [ -f "$dir/.node-version" ]; then
-  echo "Setting up node.js and yarn in $dir"
-else
-  echo "${RED}src/dev/ci_setup/setup.sh must be run within a kibana repo${C_RESET}"
-  exit 1
-fi
-
-
-export KIBANA_DIR="$dir"
-export XPACK_DIR="$KIBANA_DIR/x-pack"
-
-parentDir="$(cd "$KIBANA_DIR/.."; pwd)"
-export PARENT_DIR="$parentDir"
-
-kbnBranch="$(jq -r .branch "$KIBANA_DIR/package.json")"
-export KIBANA_PKG_BRANCH="$kbnBranch"
+source src/dev/ci_setup/setup_env.sh true
 
 echo " -- KIBANA_DIR='$KIBANA_DIR'"
```

Comment:
I think these should follow `setup_env.sh` no?

Reply:
setup_env gets called several times throughout the build, setup only gets called once. I left them in here just to cut down on noise. Think it's worth moving them?